routing_common/car_model_coefs.hpp

```diff
@@ -70,6 +71,7 @@ HighwayBasedSpeeds const kHighwayBasedSpeeds = {
 
     // The rest:
     {HighwayType::HighwayService, InOutCitySpeedKMpH(15.00 /* in city */, 15.00 /* out city */)},
+    {HighwayType::HighwayServiceEmergency, InOutCitySpeedKMpH(15.00 /* in city */, 15.00 /* out city */)},
```

Comment:
So, if we set the same weight for the service-emergency type (like for the simple service type), no need to add a separate type.